client/src/components/screens/Patient/components/FamilyTab/components/AddParentModal.tsx

```diff
@@ -0,0 +1,135 @@
+        .map((hit: any) => ({ id: hit._id, lastName: hit._source.lastName, firstName: hit._source.firstName })));
+    } else {
+      setSearchResult([]);
+    }
+  }
+
+  async function onSelectPatient(value: string, option: any) {
+    const response: any = await api.getPatientById(option.key);
+    setSelectedPatient(response.payload.data.data);
+  }
+
+  async function onSubmit() {
+    dispatch(addParentToFamily(selectedPatient?.id, parentType));
+    onClose();
+  }
+
+  return (
+    <Modal
+      visible={!!parentType}
+      title={intl.get(`screen.patient.details.family.modal.title.${parentType?.toLowerCase()}`)}
+      className="family-tab__details__add-parent__modal"
+      okText={intl.get('screen.patient.details.family.modal.add')}
+      cancelText={intl.get('screen.patient.details.family.modal.cancel')}
+      okButtonProps={{ disabled: !selectedPatient }}
+      onCancel={onClose}
+      onOk={() => form.submit()}
+    >
+      <Form
+        form={form}
+        className="family-tab__details__add-parent__modal__form"
+        onFinish={onSubmit}
+      >
+        <Form.Item
+          labelCol={{ span: 24 }}
+          label={intl.get('screen.patient.details.family.modal.search.label')}
+          className="family-tab__details__add-parent__modal__form__search"
+        >
+          <AutoComplete
+            placeholder={intl.get('screen.patient.details.family.modal.search.placeholder')}
+            notFoundContent={searchResult != null && intl.get('screen.patient.details.family.modal.search.empty')}
+            allowClear
+            onSearch={search}
+            options={searchResult?.map((sr) => ({ key: sr.id, value: `${sr.lastName.toUpperCase()} ${sr.firstName}` }))}
+            onSelect={onSelectPatient}
+          />
+        </Form.Item>
+        { !!selectedPatient && (
+          <>
+            <div className="family-tab__details__add-parent__modal__patient-card">
+              <div className="family-tab__details__add-parent__modal__patient-card__info">
+                <span className="family-tab__details__add-parent__modal__patient-card__info__name">
+                  { `${selectedPatient.lastName.toUpperCase()} ${selectedPatient.firstName}` }
+                </span>
+              </div>
+              <dl>
+                <dt>
+                  { intl.get('screen.patient.details.family.modal.ramq') }
+                </dt>
+                <dd>{ selectedPatient.ramq }</dd>
+                <dt>
+                  { intl.get('screen.patient.details.family.modal.birthDate') }
+                </dt>
+                <dd>{ selectedPatient.birthDate || '--' }</dd>
+                { !!selectedPatient.mrn && (
+                  <>
+                    <dt>
+                      { intl.get('screen.patient.details.family.modal.file') }
+                    </dt>
+                    <dd>{ `${selectedPatient.mrn[0].number} | ${selectedPatient.mrn[0].hospital}` }</dd>
```

Comment:
This will always be `undefined`. The real type of selectedPatient is `PatientData`, which means that the `mrn` attribute is of type `string[]`. 
You could replace this line with ```{ `${selectedPatient.mrn[0]} | ${selectedPatient.organization.id}` }```